examples/timer_each.py

```diff
@@ -21,7 +21,7 @@ def annoy(agent, say, more=None):
     apple.connect(addr, handler=log_message)
 
     # Multiple timers with parameters
-    orange.each(1., annoy, 'Hey')
+    orange.each(1.0, annoy, 'Hey')
```

Comment:
This is unnecessary, but if it annoys you...

Be prepared to find a couple of them in the code though. :stuck_out_tongue_winking_eye:

Reply:
I changed it thinking it was an error. Even though, the syntax is valid, this still looks better in my opinion. I can live with it, though

Reply:
Leave it as `1.0` if you want. Might be clearer for newcomers as well. :+1: